scripts/hostcfgd

```diff
@@ -1016,7 +1016,7 @@ class PasswHardening(object):
         # Use rename(), which is atomic (on the same fs) to avoid empty file
         with open(PAM_PASSWORD_CONF + ".tmp", 'w') as f:
             f.write(pam_passwh_conf)
-        os.chmod(PAM_PASSWORD_CONF + ".tmp", 0o644)
+        os.chmod(PAM_PASSWORD_CONF + ".tmp", 0o600)
```

Comment:
>0o600 [](http://example.com/codeflow?start=45&length=5)

I checked the permission for `/etc/pam.d/common-password` is 644. So suggest chmod to the same permission on .tmp file.